### DISCUSSION TITLE
Using Lowdb in Production

### DISCUSSION THREAD

**Discussion by bparanj** [vercel/vercel#6411] [Help]
Is it possible to use lowdb : https://github.com/typicode/lowdb in production? Seems like it will be part of the source code when it is deployed.

**Comment by barelyhuman** [comment]
if you bundle the file through , as in the file is a part of the source code then it will be readable, though writing to it might not be possible as the containers that the code is run in is immutable. 

From the vercel docs
> Once the build step has completed successfully, a new, immutable deployment will be made available at the preview URL.

as for if you need something temporary to be store while the service is being run you can kinda use `/tmp` to store the lowdb file but that's not persisted across deployments so you might be better off connecting to a database as a service for persisted data.
